src/content/blog/astro-210.mdx

```diff
@@ -0,0 +1,169 @@
+
+### Assets folder
+
+The new `src/assets/` folder is the recommended place to put images. You can then access them through a newly provided alias:
+
+```astro
+---
+import penguin from "~/assets/penguin.png"
+---
+```
+
+Images in this folder are also available to be optimized through content collections (explained below).
+
+### Image component
+
+The `<Image />` component has been migrated from the `@astrojs/image` integration. A typical usage of the image component looks like this:
+
+```astro
+---
+import { Image } from "astro:asset"
+import penguin from "~/assets/penguin.png"
+---
+
+<Image src={penguin} quality="high" />
+```
+
+As with the integration, you can adjust the size of the image using the `width` and `height` attributes. If not provided the image will be optimized for its actual dimensions. Additionally, the new `quality` attribute allows for adjusting the quality scale of the image.
+
+### Content Collections usage
+
+A new `image()` helper is used to define a property on your schema as being an image. Provided that the image is in the `src/assets/` folder, it will be automatically checked for you.
+
+```ts
+import { image, defineCollection, z } from "astro:content"
+
+const blogCollection = defineCollection({
+	schema: z.object({
+		title: z.string(),
+		cover: image().refine((img) => img.width === 600, {
+			message: "Cover needs to be 600 pixels wide!",
+		}),
+	}),
+})
+
+export const collections = {
+	blog: blogCollection,
+}
+```
+
+And then provide this property in any of your items in the collection:
+
+```md
+---
+title: Penguins are cool
+image: emperor-penguin.png
+---
+
+# Migration patterns of emperor penguins
+
+Post here...
+```
+
+Using this schema is a great way to ensure that your images in a collection are all of the same dimensions.
+
+## Markdoc integration
+
+<BlogContentImage src={markdoc} alt="The logo for the Markdoc project" />
+
+As we look for ways to scale Astro sites to hundreds of thousands of pages we are continuously looking for ways to make our build times faster.
+
+[Markdoc](https://markdoc.dev/) is a promising new format by Stripe that was purpose-built to scale to larger sites. Markdoc is a format that builds on top of Markdown, allowing you to build interactive content pages without sacrificing performance.
+
+Markdoc achieves this by having a declarative approach. This forces a separation between code and content, which sacrifices power and flexibility for performance.
+
+As part of the `@astrojs/markdoc integration`, we built a performance benchmark to compare Markdoc build performance at scale against our existing content formats, Markdown and MDX. In our benchmarking, we have seen up to a 3x improvement over MDX rendering static content with islands of interactivity. The results, along with a complete explanation of the findings can be found [here](https://gist.github.com/bholmesdev/07ec840d0bcdc24c0114268e16b2d0b0).
```

Comment:
```suggestion
As part of the `@astrojs/markdoc integration`, we built a performance benchmark to compare Markdoc build performance at scale against our existing content formats: Markdown and MDX. These tests show exciting improvements, with **up to 3x faster builds** over MDX rendering static content and islands of interactivity. You can find full benchmark details and findings [on our Markdoc RFC.](https://github.com/withastro/roadmap/pull/508)
```

Tweaked to highlight how exciting this metrics are, and updated the link to point to our RFC instead of a personal gist. Posting there shortly!